### ISSUE TITLE
Enums are generated incorrectly for value with ' 

### ISSUE THREAD

**Issue by retyui** [dotansimha/graphql-code-generator#5360]
**Describe the bug**

TS codegen don't escape `'` symbols in enum values (see sandbox)

```ts
export enum Foo {
  DoNot = 'Don't work' // invalid ts syntax
}
```

**To Reproduce**

https://codesandbox.io/s/billowing-platform-x7l8v?file=/types.ts

1. My GraphQL schema:

```js
const schema = new GraphQL.GraphQLSchema({
  query: new GraphQL.GraphQLObjectType({
    name: "Query",
    fields: {
      foo: {
        type: new GraphQL.GraphQLEnumType({
          name: "Foo",
          values: {
            DoNot: {
              value: "Don't work" // Hello am a buggy case
            }
          }
        })
      }
    }
  })
});
```

3. My `codegen.yml` config file:

```js
const config = {
  filename: outputPath,
  schemaAst: schema,
  plugins: [
    // Each plugin should be an object
    {
      typescript: {
        declarationKind: "interface"
      }
    }
  ],
  pluginMap: {
    typescript: typescriptPlugin
  }
}
```

**Expected behavior**

Should generate valid ts enum

```
export enum Foo {
  DoNot = 'Don\'t work' // the sterilizer should escape ' symbol
  DoNot = "Don't work" // or use " symbol for strings that includes ' symbols
}
```

**Environment:**

- OS: linux
- `@graphql-codegen/...`: latest
- NodeJS: 14

**Additional context**

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/5432

**Comment by dotansimha** [comment]
Fix available in `@graphql-codegen/typescript@1.20.1`
